src/testers.jl

```diff
@@ -122,7 +127,12 @@ function test_frule(
             _test_inferred(frule_f, deepcopy(config), deepcopy(tangents), deepcopy(primals)...; deepcopy(fkwargs)...)
         end
 
-        res = frule_f(deepcopy(config), deepcopy(tangents), deepcopy(primals)...; deepcopy(fkwargs)...)
+        # test that rules work for other tangents
+        for tangent_transform in tangent_transforms
+            call_on_copy(frule_f, config, tangent_transform.(tangents), primals...)
```

Comment:
maybe
```suggestion
            @test_msg "Shouldn't error on $tangent_transform tangents" call_on_copy(frule_f, config, tangent_transform.(tangents), primals...) isa Any
```
though idk if `@test_msg` works right on things that throw errors rather than failing